telemetry/definitions/commonDefinitions.json

```diff
@@ -21,14 +21,15 @@
             "description": "A generic version metadata"
         },
         {
-            "name": "hasException",
-            "type": "boolean",
-            "description": "Determines if the operation had an exception or not"
+            "name": "reason",
+            "type": "string",
```

Comment:
`reason` is typed in VS Code for `createSamApp`, should that be the case here too?: https://github.com/aws/aws-toolkit-vscode/blob/master/src/lambda/commands/createNewSamApp.ts#L58

Reply:
No because this can be used for other things

Reply:
Will those other things have similar reasons/can we just keep bulking up the type?

Reply:
I'd rather not build a huge enum of reasons, this field is more of a get more information field (that could contain IDE specific information) than a field that we aggregate together across IDEs